### ISSUE TITLE
Exception raised while shutting down pytest + Flask

### ISSUE THREAD

**Issue by mariokostelac** [honeycombio/beeline-python#32]
I am working on flask application. Intermittently, after tests succeed (run with flask), I get following error:
```
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/Users/mario/src/project/appenv/lib/python3.7/site-packages/libhoney/transmission.py", line 113, in _sender
    ev = self.pending.get(timeout=self.send_frequency)
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/queue.py", line 178, in get
    raise Empty
_queue.Empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/threading.py", line 917, in _bootstrap_inner
    self.run()
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/threading.py", line 865, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/mario/src/project/appenv/lib/python3.7/site-packages/libhoney/transmission.py", line 126, in _sender
    pool.submit(self._flush, events)
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/concurrent/futures/thread.py", line 151, in submit
    raise RuntimeError('cannot schedule new futures after shutdown')
RuntimeError: cannot schedule new futures after shutdown
```

beeline is initialised with an empty string as key in test env.

**Comment by tredman** [comment]
Thanks for the report! I'm opening an issue in libhoney-py, as this is where the error originates. This looks like some kind of race condition where events are being sent during shutdown, but I'm stumped at how we're getting there. At the very least we can catch the exception and prevent it from leaking into the application.

**Comment by tredman** [comment]
Hey - this is fixed in version 1.6.1 of `libhoney`.

**Comment by mariokostelac** [comment]
Thanks. When is that change going to be propagated to this package?
